app/controllers/FurloughEndDateController.scala

```diff
@@ -47,31 +48,35 @@ class FurloughEndDateController @Inject()(
     extends BaseController with I18nSupport {
 
   def onPageLoad(): Action[AnyContent] = (identify andThen getData andThen requireData).async { implicit request =>
-    getRequiredAnswersV(ClaimPeriodEndPage, FurloughStartDatePage) { (claimEndDate, furloughStart) =>
-      val preparedForm = request.userAnswers.getV(FurloughEndDatePage) match {
-        case Invalid(_)   => form(claimEndDate, furloughStart)
-        case Valid(value) => form(claimEndDate, furloughStart).fill(value)
-      }
+    getRequiredAnswerV(FurloughStartDatePage) { furloughStart =>
+      getRequiredAnswersV(ClaimPeriodStartPage, ClaimPeriodEndPage) { (claimStart, claimEnd) =>
```

Comment:
not related to this PR but we don't need to do call *V methods and do the validation incase of GETs  - because we are just reading the data stored in database already - i may be wrong but just a thought